documentation/specs/Banner.md

```diff
@@ -0,0 +1,75 @@
+# `Banner` Component Specification
+
+## Overview
+
+The `Banner` component displays a prominent message above the navigation bar at the top of the page that cannot be dismissed.
+
+### Use Cases
+
+The primary use case of the `Banner` component is to inform the user of new features, promotions, and other offerings. **Not** to be used to call attention to the user as a result of interacting with the UI.
+
+---
+
+## Design
+
+### API
+
+```typescript
+export type BannerStatus = "primary" | "success" | "neutral";
```

Comment:
**thought** i have primary here, but wondering if promotional would make more sense, not sure

Reply:
i think what you have here makes sense because it matches the states we have for `Button`s. i think that consistency will be important for consumers

<img width="846" alt="image" src="https://user-images.githubusercontent.com/752942/232081755-920cd6c7-829e-41f8-b135-7634faec80aa.png">